_gsdocs-proposals/2020/proposal_CVMFSPodmanIntegration.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Podman CVMFS integration
+layout: gsdocs_proposal
+project: CVMFS
```

Comment:
The project file _gsocprojects/2019/project_CernVM-FS.md should be copied into:  _gsocprojects/2020/project_CernVM-FS.md and referenced here in the "project:" field as CernVM-FS
The 2020/project_CernVM-FS.md should be edited for the field: cernvmfs-logo.png which should become: CERNVM-FS-logo.png (new rule). You should make a copy of images/cernvmfs-logo.png into images/CERNVM-FS-logo.png